File: Lecture 5: Bitwise Operators, For Loops, Operator Precedence & Variable Scoping/forSum.java
import java.util.Scanner;

class forSum {
    public static void main(String[] args) {

        int n;

        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the value of n: ");
        n = sc.nextInt();
        sc.close();

        int sum = 0;

        for (int i = 1; i <= n; i++) {
            sum += i;
        }

        System.out.println("Total Sum: " + sum);
    }
}

File: 005_Bitwise_Operators,_For_Loops,_Operator_Precedence_and_Variable_Scoping/forSum.cpp
#include<iostream>
using namespace std;

int main() {

    int n ;
    cout<<" enter the value of n" <<endl;
    cin >> n;

    int sum = 0;
    
    for(int i=1; i<=n; i++ ) {
        //sum = sum + i;
        sum += i;
    }

    cout<< sum <<endl;
    return 0;
}